core/logging_api.php

```diff
@@ -46,6 +46,8 @@
 	LOG_WEBSERVICE => 'WEBSERVICE'
 );
 
+$t_call_once = True;
```

Comment:
Use 'true' instead of 'True'.  Use static variable for the method rather than a global variable.  Static variables have 's_' prefix, global should have 'g_' prefix.